### DISCUSSION TITLE
React Nice Avatar Support in Laravel 🥳

### DISCUSSION THREAD

**Discussion by alissaestelle** [dapilab/react-nice-avatar#58] [Show and tell]
RNA is better than most other avatar plugins by a long shot, so I really wanted to use this package in my Laravel blog project. After some initial configuration, I was able to render avatars for each fake user in the blog comments section. My only problem was that I was rendering the same avatar for every user 😅 Luckily I was able to get that sorted after reworking the logic in my mapping function.

Anyways, here's a little demo!

https://github.com/dapi-labs/react-nice-avatar/assets/100214768/4508136d-c488-4e57-b7a3-f10009b67955

I'm refreshing the page offscreen in the video so you can see that random avatars are generated on page load (I'm not currently saving avatars to specific users in my database, but that's in the works). 

I did notice though that randomly-generated avatars excluded the pretty gradient backgrounds 💔 so I created a special gradients array and wrote logic to either set the background color as a randomly selected gradient or stick with the original color provided (that way backgrounds aren't all one or the other). Anyways ... voilà 😛

**Comment by wwayne** [comment]
Good to see that 😄 
BTW, you can try adding `isGradient` to the component to generate a gradient background, the default value if false, so it only generate simple color

**Reply by alissaestelle** [reply]
Oh amazing! I probably missed that while I was looking things over but I will definitely give it a try 👍
